packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -110,6 +112,25 @@ class Scrollbar extends StatelessWidget {
   /// null, the default value is 12.0 pixels.
   final double? hoverThickness;
 
+  /// Controls the presentation style of the scrollbar button.
+  ///
+  /// If this property is null, then [ScrollbarThemeData.buttonStyle] of
+  /// [ThemeData.scrollbarTheme] is used. If that is also null, the buttons will
+  /// not show by default.
+  ///
+  /// See also:
+  ///
+  ///  * [ScrollbarButtonStyle], the presentation style of the scrollbar button.
+  final MaterialStateProperty<ScrollbarButtonStyle?>? buttonStyle;
```

Comment:
Usually, the button is displayed together with the track, because trackVisibility is of this type, so I designed it like this here. Any better suggestions? For example, when people set the track to be visible when hovered, the button also needs to be set like this.